raiden/network/transport/matrix.py

```diff
@@ -550,7 +550,7 @@ def send_queue(address, events):
 
         for (address, _queue_name), events in queueids_to_queues.items():
             # TODO: Check if we need to separate this by queue_name
-            gevent.spawn(send_queue, address, events)
+            send_queue(address, events)
```

Comment:
As you're not spawning a greenlet, may I ask for you to just inline the body of `send_queue` here?